packages/flutter_tools/test/integration/test_driver.dart

```diff
@@ -93,6 +97,32 @@ abstract class FlutterTestDriver {
     _stderr.stream.listen(_debugPrint);
   }
 
+  Future<void> connectToVmService({bool pauseOnExceptions = false}) async {
+    _vmService = await vmServiceConnectUri(_vmServiceWsUri.toString());
```

Comment:
Missed this during the review, but the indentation of this method is off

Reply:
i'll take care of it, i have a patch that has a huge merge conflict with this anyway